src/routes/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Routes as RRDRoutes, Route } from 'react-router-dom';
+import NotFound from 'pages/404';
+import Home from 'pages/Home';
+import CustomerLeads from 'pages/CustomerLeads';
+import SingleUser from 'pages/Users/SingleUser';
+import Users from 'pages/Users';
```

Comment:
It seems this should be pages/users or the users directory should be capitalized.